src/tox_lsr/test_scripts/bootc-buildah-qcow.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+# Build a qcow2 image from a bootc buildah container
+# resulting image will be in tmp/<buildah id>/qcow2/disk.qcow2
+# Usage: build-buildah-qcow.sh <buildah id>
+set -eu
+
+BUILDAH_ID="$1"
+
+MYDIR=$(dirname $0)
+OCI_TAG="localhost/bootc-tmp:$BUILDAH_ID"
+
+# extract private SSH key from standard-inventory-qcow2, and generate pubkey from it
+# keep this in sync with ./runqemu.py
+INVENTORY_URL="https://pagure.io/fork/rmeggins/standard-test-roles/raw/linux-system-roles/f/inventory/standard-inventory-qcow2"
+INVENTORY="${TOX_WORK_DIR:-.tox}/standard-inventory-qcow2"
+[ -e "$INVENTORY" ] || curl --fail -o "$INVENTORY" "$INVENTORY_URL"
+
+PUBKEY=$(sed -n '/BEGIN.*PRIVATE KEY/,/END.*PRIVATE KEY/ { s/^.*"""//; p }' \
+    "$INVENTORY" | ssh-keygen -y -f /dev/stdin)
+
+# user's or system podman storage location
+STORAGE=$(podman info -f '{{.Store.GraphRoot}}')
+
+OS_RELEASE=$(buildah run "$BUILDAH_ID" cat /etc/os-release)
+
+# buildah → container image
+buildah commit "$BUILDAH_ID" "$OCI_TAG"
+# always clean up the temporary tag
+trap "podman rmi $OCI_TAG" EXIT INT QUIT PIPE
+
+# invoke booc-image-builder
+rm -rf tmp
```

Comment:
If this script is only intended to be run in the `tox` environment, then please use `$LSR_TOX_ENV_TMP_DIR` instead of `tmp` for the temp directory.

Reply:
But we need that path at the top level in the GitHub workflow. We can parameterize it (I didn't give it much thought, as the location is completely uninteresting for the experimentation phase at least). This will require changes on both sides and some careful testing, will do that on Monday. See also your [concern on the consumer side](https://github.com/linux-system-roles/sudo/pull/58#discussion_r2114288139).